### PR TITLE
blog: Add API Versioning post

### PR DESCRIPTION
**Pull request by nfrankel**
Screenshot 2023-11-06 at 17 57 00

### CODE REVIEW

**Review by juzhiyuan — DISMISSED** [review]
The structure looks good to me. But I didn't test this thoroughly.

File: blog/en/blog/2023/11/09/api-versioning.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
+---
+title: Resizing images on-the-fly
```
Comment (kayx23) [inline blog/en/blog/2023/11/09/api-versioning.md]: Why this title

Comment (nfrankel) [reply blog/en/blog/2023/11/09/api-versioning.md]: Because I copy-pasted and forgot to update. Good catch, thanks!

**Review by kayx23 — APPROVED** [review]
LGTM.

Also just sharing this API versioning strategy from [Stripe](https://stripe.com/blog/api-versioning) that looks interesting:

> At Stripe, we implement versioning with rolling versions that are named with the date they’re released (for example, 2017-05-24). Although backwards-incompatible, each one contains a small set of changes that make incremental upgrades relatively easy so that integrations can stay current.

**Comment by nfrankel** [comment]
Thanks for the link @kayx23. Actually, they implement the custom header-based versioning approach:

>The first time a user makes an API request, their account is automatically pinned to the most recent version available, and from then on, every API call they make is assigned that version implicitly. This approach guarantees that users don’t accidentally receive a breaking change and makes initial integration less painful by reducing the amount of necessary configuration. Users can override the version of any single request by manually setting the `Stripe-Version header`, or upgrade their account’s pinned version from Stripe’s dashboard.

**Review by yzeng25 — APPROVED** [review]

**Pull request merged**